lane.py

```diff
@@ -1114,6 +1075,15 @@ def initialize(self, library, display_name=None, genres=None,
         :param media: Only Works in one of these media will be included
         in lists.
 
+        :param fiction: Only Works with this fiction status will be included
+        in lists.
+
+        :param target_age: Only Works targeted at readers in this age range
+        will be included in lists.
```

Comment:
Previously you couldn't create a `WorkList` with a fiction or target age restriction -- not for any particular reason, it's just that only `Lane` needed that functionality. But now all the database-specific code goes into `DatabaseBackedWorkList`, and I wanted all the tests of that code to go into a class that didn't rely on creating `Lane`s at the crucial moment.

It turns out we still need to create a few `Lane`s in the test, because there's one feature that `Lane` has that `WorkList` is hard-coded not to have: a parent and the ability to inherit restrictions from the parent. But we can deal with that later.

The `license_datasource` restriction is brand new.